### PR TITLE
Added Bengali Language Support for HOWTO

### PR DESCRIPTION
**Pull request by sanjib-sen**
## What does this PR do?
Add support for the Bengali Language in HOWTO

### Why is this valuable (or not)?
This is valuable because Bengali is the 7th most spoken language in the world (265 million speakers) and a lot of Native Bengali speakers from Bangladesh, India, and Sierra Leone can use this repository to learn and play with Github Pull Requests. 

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/master/CONTRIBUTING.md)
- [x] Search for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction)

### CODE REVIEW

**Comment by davorpa** [comment]
Sorry for conflicts after merge #6542

**Comment by sanjib-sen** [comment]
Applied Requested changes in recent commit

**Comment by eshellman** [comment]
Thanks, and welcome to free-programming-books!

File: HOWTO-bn.md [hunk 1]
```diff
@@ -2,12 +2,12 @@
...
 Free-Programming-Books রিপোজটরি তে স্বাগতম! আমরা নবাগত কন্ট্রিবিউটরস দের স্বাগতম জানাই; এমনকি যারা গিটহাবে এই প্রথম কোন "পুল রিকোয়েস্ট" তৈরি কয়েছেন। যদি আপনি তাদের একজন হয়ে থাকেন তাহলে নিচের রিসোর্স গুলো আপনার কাজে লাগতে পারেঃ
 
-* [পুল রিকোয়েস্ট কি?](https://help.github.com/articles/about-pull-requests/)
-* [কিভাবে পুল রিকোয়েস্ট দিব](https://docs.github.com/en/free-pro-team@latest/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request)
-* [গিটহাব হ্যালো ওয়ার্ল্ড](https://guides.github.com/activities/hello-world/)
-* [ইউটিউব - নতুনদের জন্য গিটহাব](https://www.youtube.com/watch?v=0fKg7e37bQE)
-* [ইউটিউব - কিভাবে একটি গিটহাব রিপোজিটরি ফোর্ক করবেন এবং পুল রিকোয়েস্ট সাবমিট করবেন](https://www.youtube.com/watch?v=G1I3HF4YWEw)
-* [ইউটিউব - মার্কডাউন ক্র্যাশ কোর্স ](https://www.youtube.com/watch?v=HUBNt18RFbo)
+* [:us: পুল রিকোয়েস্ট কি?](https://help.github.com/articles/about-pull-requests/) *(in english)*
+* [:us: কিভাবে পুল রিকোয়েস্ট দিব](https://docs.github.com/en/free-pro-team@latest/github/ collaborating-with-issues-and-pull-requests/creating-a-pull-request) *(in english)*
```
Comment (davorpa) [inline HOWTO-bn.md:6]: There are an extra space in the middle of URL

Comment (davorpa) [reply HOWTO-bn.md:6]: Fixed at https://github.com/EbookFoundation/free-programming-books/commit/987e0b733c8265cd743041a5fea51c6f3239346e

**Pull request merged**